packages/apps/helpdesk/src/screens/TicketFormScreen.js

```diff
@@ -0,0 +1,320 @@
+  const [responsibleUser, setResponsibleUser] = useState(
+    ticket?.responsibleUser,
+  );
+  const [_ticket, setTicket] = useState(idTicket != null ? ticket : {});
+  const [disabledButton, setDisabledButton] = useState(
+    isObjectMissingRequiredField(_ticket),
+  );
+
+  const handleTicketFieldChange = (newValue, fieldName) => {
+    setTicket(current => {
+      if (!current) {
+        return {[fieldName]: newValue};
+      }
+      current[fieldName] = newValue;
+      return current;
+    });
+    setDisabledButton(isObjectMissingRequiredField(_ticket));
+  };
+
+  useEffect(() => {
+    dispatch(fetchTicketById({ticketId: idTicket}));
+  }, [dispatch, idTicket]);
+
+  useEffect(() => {
+    if (client?.id != null) {
+      dispatch(getCustomerbyId({customerId: client?.id}));
+    }
+  }, [dispatch, client?.id]);
+
+  const handleChangeValueProject = value => {
+    setProjectInput(value);
+    setClient(value?.clientPartner);
+    setContactPartner(value?.contactPartner);
+  };
+
+  const updateTicketAPI = useCallback(() => {
+    dispatch(
+      updateTicket({
+        ticket: {
+          ..._ticket,
+          ticketType: _ticket?.ticketType
+            ? {id: _ticket?.ticketType?.id}
+            : null,
+          project: projectInput ? {id: projectInput?.id} : null,
+          customerPartner: client ? {id: client?.id} : null,
+          contactPartner: contactPartner ? {id: contactPartner.id} : null,
+          assignedToUser: assignedTo ? {id: assignedTo?.id} : null,
+          responsibleUser: responsibleUser ? {id: responsibleUser?.id} : null,
+          prioritySelect: priority,
+        },
+      }),
+    );
+    navigation.navigate('TicketDetailsScreen', {
+      idTicket: _ticket.id,
+    });
+  }, [
+    _ticket,
+    dispatch,
+    navigation,
+    contactPartner,
+    client,
+    projectInput,
+    assignedTo,
+    responsibleUser,
+    priority,
+  ]);
+
+  return (
+    <Screen>
+      <KeyboardAvoidingScrollView style={styles.scroll}>
+        <View style={styles.container}>
+          <FormInput
+            style={styles.input}
+            title={I18n.t('Hepdesk_Subject')}
+            onChange={value => handleTicketFieldChange(value, 'subject')}
+            defaultValue={_ticket.subject}
+            required={true}
+          />
+          <ProjectSearchBar
+            titleKey="Helpdesk_Project"
+            placeholderKey="Helpdesk_Project"
+            defaultValue={projectInput}
+            onChange={handleChangeValueProject}
+            style={styles.picker}
+            styleTxt={styles.marginTitle}
+          />
+          <FormIncrementInput
+            title={I18n.t('Helpdesk_Progress')}
+            defaultValue={_ticket.progressSelect?.toString()}
+            onChange={value => handleTicketFieldChange(value, 'progressSelect')}
+            decimalSpacer={I18n.t('Base_DecimalSpacer')}
+            thousandSpacer={I18n.t('Base_ThousandSpacer')}
+          />
```

Comment:
Progress is not an decimal value tike others. You can test on the web instance if you want, we can only set particular interger value so I think we should developp a specific component in Helpdesk module : ProgressFormInput which should have the same design as FormIncrementInput but formats the value entered to the nearest decimal place, and an attribute should be added to the Increment component to disable automatic formatting (formatting must be enabled by default).